examples/specs/bubble_health_income.json

```diff
@@ -0,0 +1,23 @@
+{
+  "description": "A bubble plot showing the correlation between health and income for 187 countries in the world.",
```

Comment:
@ZeningQu your call. The data is originally from http://www.gapminder.org/data/, right?